.github/workflows/clang-tidy-linux.yml

```diff
@@ -8,7 +8,7 @@ on:
       - '.github/workflows/clang-tidy-linux.yml'
```

Comment:
kind of remember that this will trigger a new clang-tidy build, if it has a new SHA, might want to double check the pytorch/pytorch's reference to it just in case

Reply:
If I land it before rename, it shouldn't should it? Worst case scenario, I restore previous version on AWS